### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by aracelicaes**
Pull request milestone 3.

### CODE REVIEW

File: lib/decision_maker.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  def position_remover(position)
+    @moves.delete(position)
+  end
+
+  # checks for win
+  def check_for_winner
+    answer = false
+    WIN_POSSIBILITIES.each do |possibilities|
+      case @board.board.values_at(*possibilities)
+      when %w[🔵 🔵 🔵]
```
Comment (nikasulo) [inline lib/decision_maker.rb:76]: Can we find a way to do this more neatly/performantly without looping through the `WIN_POSSIBILITIES` on every turn?

File: lib/decision_maker.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  def board_get
+    @board = Board.new
+  end
+
+  # once games starts requests 1st move until no more spots are avlbl on board
+  def move_sequence
+    winner = nil
+    until winner || @moves.empty?
+      puts @board.the_actual_board
```
Comment (nikasulo) [inline lib/decision_maker.rb]: Please note that there should be no `puts`/`get.chomp`/`print` statements outside of the `main.rb` file. All other files should contain strictly logic and no display code. Please see requirement below:
screenshot

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work team! :rocket: 
Your work is really beautiful! I love the interface, it is very thoughtful and unique. Please consider my comments below towards aligning more with the requirements for this project, I wish you the very best and see you at the top! :medal_military: 

#### Gemfile
Your gemfile is missing, it should at the very least, include a URL for gem sources and the `tty-prompt` gem listed. It's absence prevented me from installing/running the game smoothly. I had to create the file and list the gems myself.

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: lib/decision_maker.rb (hunk 1 above)
Comment (nikasulo) [reply lib/decision_maker.rb:76]: This negatively impacts the speed of your logic

**Review by Forison — APPROVED** [review]
## Status:: Approved✔️
# 👋Team!!!
## Good on addressing the issues raised in the previous review,
### Go ahead and merge the branch and proceed to your next task😎 
#### Happy coding🎲
Addo

**Pull request merged**